Extensions/pokes.css

```diff
@@ -1,73 +1,215 @@
 }
 
-#xkit-loading_pokemon {
+/*
+ * Pokémon list style 
+*/
+.xkit-pokes-lightbox {
+	position: fixed;
+	top: 0px;
+	bottom: 0px;
+	right: 0px;
+	left: 0px; 
+	z-index: 15000;
+	overflow: hidden;
+	background-color: rgba(25,25,25,.92);
+}
+.gender {
+	display: none;
+	float: right;
+	font-family: "segoe ui symbol";
+	font-size: 20px;
+}
+.nickname {
+	display: none;
+	font-weight: 700;
+	font-size: 25px;
+}
+.nickname::before {
```

Comment:
The thing is display: none that keeps it from showing up. The icon is supposed to be the edit pen that tumblr uses on posts you can edit if you know what I mean. I did not implement nicknaming yet so I decided to hide the symbol until Nicknames become a thing